From cac575e606841c450b7c46f1dd55f94d60e99e03 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 14 Nov 2024 10:01:52 +0100
Subject: [PATCH] arm64: dts: st: add channels in mdf nodes stm32mp257f-dk

Add channels child nodes in MDF filter nodes on STM32MP257F DK.

Change-Id: I7c00b68ace716e7a98ae198bae6f07ab83c4151f
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/420092
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index a6e555c90d6d..40cc86d9e401 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -619,11 +619,18 @@ sitf6: sitf@380 {
 	};
 
 	filter0: filter@84  {
+		#address-cells = <1>;
+		#size-cells = <0>;
 		st,cic-mode = <4>;
 		st,sitf = <&sitf6 0>;
 		st,hpf-filter-cutoff-bp = <625>;
 		status = "okay";
 
+		channel@0 {
+			reg = <0>;
+			label = "dmic_u53";
+		};
+
 		asoc_pdm0: mdf-dai {
 			compatible = "st,stm32mp25-mdf-dai";
 			#sound-dai-cells = <0>;
@@ -640,11 +647,18 @@ mdf_endpoint0: endpoint {
 	};
 
 	filter1: filter@104  {
+		#address-cells = <1>;
+		#size-cells = <0>;
 		st,cic-mode = <4>;
 		st,sitf = <&sitf6 1>;
 		st,hpf-filter-cutoff-bp = <625>;
 		status = "okay";
 
+		channel@1 {
+			reg = <1>;
+			label = "dmic_u12";
+		};
+
 		asoc_pdm1: mdf-dai {
 			compatible = "st,stm32mp25-mdf-dai";
 			#sound-dai-cells = <0>;
-- 
2.50.1

